src/request/sign-transaction/SignTransaction.js

```diff
@@ -53,12 +54,13 @@ class SignTransaction {
                 : request.recipientLabel || null,
             imageUrl: request.shopLogoUrl || null,
             accountLabel: null,
-        });
+        }, request.layout === SignTransactionApi.Layouts.CASHLINK);
         this._recipientAddressInfo.renderTo($recipient);
-        $recipient.addEventListener('click', () => {
-            this._openDetails(this._recipientAddressInfo);
-        });
-
+        if (request.layout !== SignTransactionApi.Layouts.CASHLINK) {
+            $recipient.addEventListener('click', () => {
+                this._openDetails(this._recipientAddressInfo);
+            });
+        }
```

Comment:
Because we disable the opening of the account details when the layout is cashlink, the cashlink display options can be removed from `AddressInfo` again and the condition in line 57 can also be removed.